tests/bearlib/languages/documentation/documentation_extraction_testdata/doxygen.py

```diff
@@ -32,3 +32,11 @@ def best_docstring(param1, param2):
     # @param param2 Parameter 2
     # @return Nothing
         return None
+
+def docstring_find(filename):
+    ## This is dummy docstring find function!
+    #
+    # @param filename contains filename.
```

Comment:
Just make sure you code is awesome. And raise newcomer issues for other improvements